src/subscription_manager/cache.py

```diff
@@ -191,6 +195,86 @@ def update_check(self, uep, consumer_uuid, force=False):
             return 0  # No updates performed.
 
 
+class InMemoryCache(object):
```

Comment:
We should remove this as it is not used in this branch or in the main branch.

Reply:
removed